tests/console/yast2_samba.pm

```diff
@@ -230,9 +227,45 @@ sub run {
     send_key 'alt-o';
 
     wait_serial('yast2-samba-server-status-0', 60) || die "'yast2 samba-server' didn't finish";
+}
+
+sub gui_new {
+    assert_screen 'yast2_samba-workgroup';
+    send_key 'alt-n';
+    if (check_screen 'yast2_still_susefirewall2', 10) {
+        record_soft_failure 'bsc#1064405';
+        send_key 'alt-c';
+    }
+    wait_still_screen;
+    # from here we have new workflow because of changes in sles 15
+}
+
+
+sub run {
+    setup_ldap;
+
+    # install samba stuffs at first
```

Comment:
the comment is a bit redundant. Everyone should be able to understand the next line just as well

Reply:
okay, removed